contrib/buildrefactor/src/python/pants/contrib/buildrefactor/buildozer.py

```diff
@@ -5,6 +5,8 @@
 from __future__ import (absolute_import, division, generators, nested_scopes, print_function,
                         unicode_literals, with_statement)
 
+import warnings
```

Comment:
please follow the same pattern with `logger` as other pants components uses. https://github.com/twitter/pants/blob/e3fceaac9315a44ab8b07dc69938acdc395b738d/src/python/pants/binaries/binary_util.py#L41